### PR TITLE
fix(popup): fix licensing issue for create react context

### PR DESCRIPTION
**Pull request by bmdalex**
# Popup `create-react-context` license fix

This PR fixes the #263 issue by referencing correct package versions unaffected by the license file that prohibits Microsoft from using it.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -1592,7 +1592,7 @@ create-hmac@^1.1.0, create-hmac@^1.1.2, create-hmac@^1.1.4:
     safe-buffer "^5.0.1"
     sha.js "^2.4.8"
 
-create-react-context@^0.2.1:
+create-react-context@^0.2.3:
```
Comment (kuzhelov) [inline yarn.lock]: heads up, this won't fix the issue - consider to review text of LICENSE for this version, we won't be able to comply. Consider to enforce version 0.2.2 instead (or lower) - and as the newer version will be published (assuming the LICENSE would still be MIT) we will be able to update the version here.

Comment (bmdalex) [reply yarn.lock]: I don't understand , 0.2.3 has MIT license..

Comment (bmdalex) [reply yarn.lock]: oh... the license was introduced in 0.2.3...and the removal directly to master and was not released yet...

Comment (kuzhelov) [reply yarn.lock]: `create-react-context@0.2.2`

Comment (kuzhelov) [reply yarn.lock]: agreed to make changes in package.json instead, to include `resolutions` section: https://yarnpkg.com/lang/en/docs/selective-version-resolutions/

Comment (bmdalex) [reply yarn.lock]: done, @kuzhelov can u please take another look?

File: yarn.lock [hunk 2]
```diff
@@ -1592,9 +1592,9 @@ create-hmac@^1.1.0, create-hmac@^1.1.2, create-hmac@^1.1.4:
     safe-buffer "^5.0.1"
     sha.js "^2.4.8"
 
-create-react-context@^0.2.1:
-  version "0.2.3"
-  resolved "https://registry.yarnpkg.com/create-react-context/-/create-react-context-0.2.3.tgz#9ec140a6914a22ef04b8b09b7771de89567cb6f3"
+create-react-context@0.2.2, create-react-context@^0.2.1:
```
Comment (kuzhelov) [inline yarn.lock:1595]: this change seems to be unnecessary - would rather avoid touching yarn artefacts, Let me try this out..

File: package.json [hunk 3]
```diff
@@ -94,6 +94,9 @@
     "tmp": "^0.0.33",
     "what-input": "^5.1.2"
   },
+  "resolutions": {
```
Comment (kuzhelov) [inline package.json]: nit: it is better to put this section after dependencies sections (i.e. `dependencies`, `devDependencies`, `peerDependencies`) - that's the general practice

File: yarn.lock (hunk 2 above)
Comment (bmdalex) [reply yarn.lock:1595]: this was generated by yarn itself, only change I made was in `package.json`

File: package.json (hunk 3 above)
Comment (bmdalex) [reply package.json]: will change

Comment (bmdalex) [reply package.json]: done

**Review by kuzhelov — APPROVED** [review]

**Pull request merged**